### PR TITLE
update button positions to a more logical place

### PR DESCRIPTION
**Pull request by JohnnyXP64**
i couldn't put the change map button exactly where i wanted, but at least the disconnect / back and the start game, and the change map are not i a place where i think its less confusing for the players.
regards
Johnnyxp64

### CODE REVIEW

File: mods/ra/chrome/lobby.yaml [hunk 1]
```diff
@@ -78,16 +78,21 @@ Background@SERVER_LOBBY:
 			Font: Bold
 			Text: Game Options
 		Button@CHANGEMAP_BUTTON:
-			X: 336
-			Y: PARENT_BOTTOM - 291
+			#X: 336
+			X: PARENT_RIGHT - WIDTH - 60
+			#Y: PARENT_BOTTOM - 291
```
Comment (Mailaender) [inline mods/ra/chrome/lobby.yaml]: Please don't out-comment lines. Just change them.

**Comment by Mailaender** [comment]
This is a bit weird:

image

probably switch label and button positions for consistency.

**Comment by pchote** [comment]
I find this layout to be less logical than what we have now:
* Moving the back button to the bottom-left breaks the convention used in all the other panels
* Moving the start game button to the bottom-right encourages bad behavior (it is rude to force-start a game before players are ready - the proper way for a game to start is to have everyone check the ready checkboxes).
* I agree with @Mailaender about the change map button position being weird.

**Comment by JohnnyXP64** [comment]
I cant find the label object in the yaml file! That’s why I said is not exactly as I was thinking it ;-)

**Comment by JohnnyXP64** [comment]
hi,

as i was saying the chnage button should be at the label location and vise versa, but in the yaml file i didnt found a label for the map preview...so i tried this

also teh back and the "next" or go, based on application design prensipals that we have been using for decades now, shoud either be in top or bottom, but left and right positions just as i sujest.
the next logical step is to start the game and the back logica step it to go back, using windows or any OS and any braswer you will see the same logic, our brains are used to search for the back in the top LEFT or bottom LEFT corners, and the button that does the "next action" on the right Side.

if the Start Game really makes you feel that is "forcing" players to start, at least dont "hide" it where it is right now, and its easily detected.
dont forget also the skirmish players that are i think the majority on this project, we wish to start a game fast, and the mouse goes down riht, only to see that there is nothing there to start the game!

File: mods/ra/chrome/lobby.yaml (hunk 1 above)
Comment (JohnnyXP64) [inline mods/ra/chrome/lobby.yaml]: ok. noted thanks

**Comment by Unit158** [comment]
If any of these changes make the lobby inconsistent with the other panels, you must change the other panels as well. (Unless another dev is willing to do it?)

:-1: for an unneeded subjective change without a poll from the community. This is the first time I've heard or seen a complaint about the button positions in the GUIs, and we will certainly get a bunch of users grumbling about the change.

**Comment by Mailaender** [comment]
No consensus for this so let's top right here.

**Comment by JohnnyXP64** [comment]
i wouldn't have any issues do the change to all the panels if its needed. but if you have already decided without a community poll, well what can i say. too bad you don't see the issue with the unfriendly Button. 

@Unit158 because users may have been used to something wrong, doesn't make it correct thought. After all OpenRA since it started has gone under huge UI changes, eventually users did liked them.

**Comment by Phrohdoh** [comment]
I have mentioned moving the start button quite a few times but it is always a rejected notion.

**Comment by JohnnyXP64** [comment]
@Phrohdoh i started with this simple yet annoying thing, just to see the "contribution" attitude they advertise.....nope, looks more like a closed source attitude to me. i am at least disappointed.-

**Comment by Mailaender** [comment]
This is open source and openly developed, but indeed the standards to get your patches included are now quite high. I bet we can find a compromise here though.

**Comment by pchote** [comment]
@JohnnyXP64 in a project like this there is always a balance between "anyone can contribute" and "everything is accepted", especially when dealing with things like the lobby and sidebar where everyone has their own slightly-different opinion about how they should look and work.  It's important to keep a consistent UI design to avoid the too many cooks problem.

With that out of the way, I agree that the start-game workflow discoverability sucks.  A few proposals have been discussed in IRC over the years to improve this - my favorite is to make the ready checkbox blink after 5 seconds with no configuration changes (which is uncontroversial), and then remove the force-start button (which is controversial).

**Comment by JohnnyXP64** [comment]
@pchote so what you are saying is remove the start game button completely and start the game automatically when everyone clicks "ready"? how about the skirmish? don't you think an "automatic start" gives the feeling that you are not controlling the game, the game controls you?

i just think that at least the start game and the change map should be relocated cause as you nicely put "workflow discoverability sucks".

i don't see why this small (because its small) improvement takes you "years" to decide. its not a personal opinion that where it is now is wrong, everyone agrees as i see, the problem starts when we talk about where to relocate it. well i gave my suggestion, if others can suggest something better lets see it, not just saying "i don't like it subject is closed".

thats where i was disappointed and i wish to move one from that.

**Comment by Mailaender** [comment]
Sorry. That was your first pull request. It was a bit rude from me to just close it. See https://en.wikipedia.org/wiki/Parkinson's_law_of_triviality and http://www.urbandictionary.com/define.php?term=bikeshedding on the phenomenon that trivial changes attract a lot of dispute.

**Comment by LipkeGu** [comment]
@pchote we could use the TS logic :) 
- all players checked the checkbox and the games starts after 10 seconds (counting down to 0)

**Comment by JohnnyXP64** [comment]
still the start game is not the only button miss placed! :/

**Comment by abcdefg30** [comment]
The game already starts, when every player has ticked ```Ready```.

**Comment by pchote** [comment]
This hasn't seen any further discussion, so i'm closing this for now as part of a pr queue cleanup.  It would be a good idea to discuss these in IRC if you want to continue the topic.

**Pull request closed without merging**